src/ray/common/runtime_env_manager.cc

```diff
@@ -29,12 +29,19 @@ void RuntimeEnvManager::AddURIReference(const std::string &hex_id,
                                         const rpc::RuntimeEnvInfo &runtime_env_info) {
   const auto &uris = runtime_env_info.uris();
   if (!uris.working_dir_uri().empty()) {
+    uri_reference_.reserve(uri_reference_.size() + uris.working_dir_uri().size());
```

Comment:
If URI already exists, we only want to increment it and this reserve would reserve some unused mem.

Reply:
You're right, removed.